src/test/java/com/twilio/oai/TwilioGeneratorTest.java

```diff
@@ -28,14 +28,8 @@
 public class TwilioGeneratorTest {
     @Parameterized.Parameters
     public static Collection<Generator> generators() {
-        return Arrays.asList(Generator.TWILIO_CSHARP,
```

Comment:
I think these changes need to be reverted, guess they were part of testing.

Reply:
Will include other languages